index.html

```diff
@@ -197,10 +202,47 @@ <h3>Skills</h3>
             <li>Codepen</li>
             <li>Gitlab</li>
             <li>Terminal</li>
-          </ul>  
+          </ul>
         </div>
       </li>
     </ul>
   </section>
+
+  <section class="contact">
+    <p class="get-touch">
+      I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+    </p>
+    <form class="data-form" action="https://formspree.io/f/mjvlbpay" method="post">
+      <label for="contact-name">
+        <input type="text" class="contact-form" id="contact-name" name="contact-name" placeholder="Full name" maxlength="30" required>
+      </label>
+      <label for="contact-email">
+        <input type="email" class="contact-form" id="contact-email" name="contact-email" placeholder="Email address" required>
+      </label>
+      <textarea class="contact-form" id="contact-message" name="contact-message" placeholder="Enter text here" maxlength="500" required></textarea>
+      <button type="submit" id="submit">Get in touch</button>
```

Comment:
- [ ] Hi the rest of your form design matches the Figma really well 👏. Although I'd like to mention that you have missed including a background image that is Visible on the Figma template. Please implement this
## Your Design
![image](https://user-images.githubusercontent.com/18730106/144139452-79370860-5245-406e-9ef5-38b73367c7d6.png)
## Figma Design
![image](https://user-images.githubusercontent.com/18730106/144139486-d43a9d42-e86f-4dbd-9cef-90a273e9f0ac.png)